src/Mvc/ViewCompilation/test/FunctionalTests/Resources/SimpleAppTest.Home.Index.txt

```diff
@@ -7,7 +7,7 @@
 
     
     
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" />
```

Comment:
@ryanbrandenburg are these dangling differences between the `release/2.1` and `release/2.2` important? Neither Bootstrap version lines up with the version in our templates&hellip;

Either way, I'm surprised `git merge release/2.1 --no-ff -X ours` then favouring all deleted files in `git mergetool` left anything behind.

Reply:
The difference we see here seem correct. Unfortunately the 2.2 branch wasn't just a cherry-pick of the 2.1 commit because we had changed some projects from BS3.X to BS4.X, if that answers your question?